infra/main.bicep

```diff
@@ -274,6 +308,9 @@ output AZURE_OPENAI_RESOURCE_GROUP string = openAiResourceGroup.name
 output AZURE_OPENAI_GPT_DEPLOYMENT string = gptDeploymentName
 output AZURE_OPENAI_CHATGPT_DEPLOYMENT string = chatGptDeploymentName
 
+output AZURE_FORMRECOGNIZER_SERVICE string = formrecognizer.outputs.name
```

Comment:
Please keep consistent case for formRecognizer, camel

Reply:
Thanks for feedback. Addressed in PR: https://github.com/Azure-Samples/azure-search-openai-demo/pull/49